[2025-06-30T05:31:00.099+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-30T05:31:00.298+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mysql_dag.insert_data manual__2025-06-30T05:29:54.890763+00:00 [queued]>
[2025-06-30T05:31:00.318+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mysql_dag.insert_data manual__2025-06-30T05:29:54.890763+00:00 [queued]>
[2025-06-30T05:31:00.320+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-06-30T05:31:00.381+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): insert_data> on 2025-06-30 05:29:54.890763+00:00
[2025-06-30T05:31:00.391+0000] {standard_task_runner.py:63} INFO - Started process 707 to run task
[2025-06-30T05:31:00.398+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'mysql_dag', 'insert_data', 'manual__2025-06-30T05:29:54.890763+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/mysql.py', '--cfg-path', '/tmp/tmpaoslr982']
[2025-06-30T05:31:00.405+0000] {standard_task_runner.py:91} INFO - Job 49: Subtask insert_data
[2025-06-30T05:31:00.435+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-06-30T05:31:00.511+0000] {task_command.py:426} INFO - Running <TaskInstance: mysql_dag.insert_data manual__2025-06-30T05:29:54.890763+00:00 [running]> on host 76cdd04f4df6
[2025-06-30T05:31:00.631+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Karthik' AIRFLOW_CTX_DAG_ID='mysql_dag' AIRFLOW_CTX_TASK_ID='insert_data' AIRFLOW_CTX_EXECUTION_DATE='2025-06-30T05:29:54.890763+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-30T05:29:54.890763+00:00'
[2025-06-30T05:31:00.632+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-30T05:31:00.753+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.760+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.766+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.770+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.774+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.779+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.782+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.787+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.791+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.796+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.801+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.805+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.809+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.813+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.817+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.821+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.826+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.830+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.834+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.839+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.847+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.850+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.868+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.872+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.875+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.879+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.883+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.887+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.891+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.895+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.899+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.903+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.906+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.910+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.914+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.919+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.923+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.928+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.932+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.936+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.940+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.944+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.948+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.952+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.956+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.960+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.965+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.969+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.974+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.978+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.983+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:00.988+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.030+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.034+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.038+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.042+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.048+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.053+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.058+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.063+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.068+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.072+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.076+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.081+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.085+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.091+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.096+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.100+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.104+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.107+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.111+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.116+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.121+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.125+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.129+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.134+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.138+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.142+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.146+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.150+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.154+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.157+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.160+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.164+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.168+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.172+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.175+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.180+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.185+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.189+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.193+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.198+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.202+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.207+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.212+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.217+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.222+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.227+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.232+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.235+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.238+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.243+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.248+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.253+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.256+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.260+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.265+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.270+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.274+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.279+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.284+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.288+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.292+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.297+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.302+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.306+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.310+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.314+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.318+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.322+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.326+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.331+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.335+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.339+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.343+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.347+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.351+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.356+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.361+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.365+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.370+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.374+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.378+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.382+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.386+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.390+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.394+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.398+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.402+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.405+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.409+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.413+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.417+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.421+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.425+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.429+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.433+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.438+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.442+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.446+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.450+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.455+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.458+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.463+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.468+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.473+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.478+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.483+0000] {logging_mixin.py:188} INFO - error :  1305 (42000): FUNCTION ***_practice.getdate does not exist
[2025-06-30T05:31:01.485+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-06-30T05:31:01.486+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-30T05:31:01.497+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=mysql_dag, task_id=insert_data, run_id=manual__2025-06-30T05:29:54.890763+00:00, execution_date=20250630T052954, start_date=20250630T053100, end_date=20250630T053101
[2025-06-30T05:31:01.577+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-06-30T05:31:01.600+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-30T05:31:01.608+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
